src/desktop/assets/artist2.js

```diff
@@ -1 +1,38 @@
 import 'desktop/apps/artist2/client'
+import cta from 'desktop/apps/artist/client/cta.coffee'
+import metaphysics from 'lib/metaphysics.coffee'
+import Artist from 'desktop/models/artist.coffee'
+import { data as sd } from 'sharify'
+
+const query = `
+query ArtistCTAQuery($artistID: String!) {
+  artist(id: $artistID) {
+    href
+    name
+    cta_image: image {
+      thumb: resized(width: 150, version: "square") {
+        url
+      }
+    }
+    cta_artists: artists(size: 1) {
+      image {
+        thumb: resized(width: 150, version: "square") {
+          url
+        }
+      }
+      name
+    }
+  }
+}
+`
+const send = {
+  method: 'post',
+  query,
+  variables: { artistID: 'andy-warhol' },
+}
+
+if (sd.ARTIST_PAGE_CTA_ENABLED) {
+  metaphysics(send).then(({ artist }) => {
+    cta(new Artist(artist))
+  })
+}
```

Comment:
Rather than keeping this in `/assets` can you move to the `apps/artist2/components` folder?